drivers/VDI.py

```diff
@@ -864,15 +864,15 @@ def _activate_cbt_log(self, logname):
         SR specific Implementation required for VDIs on block-based SRs.
         No-op otherwise
         """
-        pass
+        raise NotImplementedError("Override in implementation")
```

Comment:
Now that we're using python3 this could have an @abstractmethod decorator to forbid instantiating classes unless this method is overriden.

Reply:
well, actually this wrong anyway as only one implementing class overrides it. It just needs to be a no-op but actually satisfy the expectation of the caller and return something